components/Movie.js

```diff
@@ -1,23 +1,48 @@
-import styles from "../styles/Movie.module.css"
-import Typography from '@material-ui/core/Typography';
+import { Card, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
 
-export const Movie = ({ Year, Title, Poster }) => {
+const styles = {
+    cardRoot: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: '200px',
+        margin: '16px',
+        padding: '18px',
+    },
+    about: {
+        display: 'flex',
+        flexDirection: 'column',
+        marginTop: '16px',
+    },
+    poster: {
+        height: '300px',
+        width: '200px',
+    },
+};
+
+const handleError = (e) => {
+    const defaultImgLocation = "/assets/default-poster.jpeg"
+
+    e.target.src = defaultImgLocation
+}
+
+const Movie = ({ Year, Title, Poster, ...props }) => {
+    const { classes } = props;
     return (
-        <div>
-            <Typography variant="h6" className={styles.title}>{Title}</Typography>
-            <Typography variant="h6">{Year}</Typography>
+        <Card className={classes.cardRoot}>
             <img
-                className={styles.poster}
+                className={classes.poster}
                 src={Poster}
                 onError={handleError}
                 alt={`Movie poster for ${Title}`}
             />
-        </div>
+            <div className={classes.about}>
+                <Typography variant="h6">{Title}</Typography>
+                <Typography variant="h6">{Year}</Typography>
+            </div>
+        </Card>
     )
 }
 
-const handleError = (e) => {
-    const defaultImgLocation = "/assets/default-poster.jpeg"
-
-    e.target.src = defaultImgLocation
-}
+export default withStyles(styles)(Movie);
```

Comment:
This is what I was talking about when I mentioned there's 3 APIs to apply styles in MUI. Is this your preferred way?
https://material-ui.com/styles/basics/

Reply:
LOL yeah I know, there's also just inline styling `style={{ color: 'red' }}` 👀 I do tend to use this way to style but hooks can't do this so idk 🤷